.evergreen/functions.yml

```diff
@@ -405,7 +406,7 @@ functions:
           <<: *compass-env
           DEBUG: ${debug}
           npm_config_loglevel: ${npm_loglevel}
-          COMPASS_DISTRIBUTION: ${compass_distribution}
+          HADRON_DISTRIBUTION: ${compass_distribution}
```

Comment:
Changed it back to HADRON_ because otherwise it's hard to search through the code for all occurences, I'm totally up for changing this to COMPASS_ at some point, but we should probably do this for all the mentions of hadron